utils/i18nUtils.js

```diff
@@ -0,0 +1,118 @@
+import i18next from "i18next";
+import { initReactI18next } from "react-i18next";
+import resourcesToBackend from "i18next-resources-to-backend";
+import { match } from "@formatjs/intl-localematcher";
+import { shouldPolyfill } from "@formatjs/intl-locale/should-polyfill";
+
+// Update this to the default locale for the app
+const DEFAULT_LOCALE = "en";
+// Update this to the supported locales contained in the `locales` folder
+const SUPPORTED_APP_LOCALES = ["en", "fr"];
+
+// TODO: Export polaris supported locales from the Polaris package
+const SUPPORTED_POLARIS_LOCALES = [
```

Comment:
Note: I'm going to reach out to the Polaris team and open a PR to export this constant from `@shopify/polaris`.